go.mod

```diff
@@ -1,9 +1,15 @@
 module github.com/flipgroup/goverdiff
 
-go 1.15
+go 1.16
 
 require (
+	github.com/golang/protobuf v1.5.1 // indirect
 	github.com/google/go-github/v33 v33.0.0
-	golang.org/x/oauth2 v0.0.0-20180821212333-d2e6202438be
+	github.com/google/go-querystring v1.1.0 // indirect
+	golang.org/x/crypto v0.0.0-20210322153248-0c34fe9e7dc2 // indirect
+	golang.org/x/mod v0.4.2 // indirect
+	golang.org/x/net v0.0.0-20210326060303-6b1517762897 // indirect
+	golang.org/x/oauth2 v0.0.0-20210323180902-22b0adad7558
 	golang.org/x/tools v0.1.0
+	google.golang.org/appengine v1.6.7 // indirect
 )
```

Comment:
Why did this list change so significantly if the main dependencies didn't!?

Reply:
I'm thinking maybe an up to date `go.mod` wasn't in `main` before I started this branch. So I'm probably just brining it up to sync.

Reply:
Probably. Is it worth bumping the GitHub dependency while we're here?

Reply:
Good call - bumped with adc6710.